### PR TITLE
update rnaz to v2.1.1

### PR DESCRIPTION
**Pull request by jfallmann**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/rnaz/meta.yaml [hunk 1]
```diff
@@ -18,7 +18,6 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - libstdcxx-ng
-    - libsvm =3.21
```
Comment (bgruening) [inline recipes/rnaz/meta.yaml]: Can you remove this strict perl pinning please?

File: recipes/rnaz/build.sh [hunk 2]
```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-./configure CFLAGS="-std=gnu89 -g -O2" --prefix=$PREFIX --datadir=$PREFIX/share
-make
+./autogen.sh
+./configure --prefix=$PREFIX --datadir=$PREFIX/share
```
Comment (eggzilla) [inline recipes/rnaz/build.sh]: Did you test if removing the -O2 flag changes the executable speed?

File: recipes/rnaz/meta.yaml (hunk 1 above)
Comment (eggzilla) [reply recipes/rnaz/meta.yaml]: Removed the pinning and it still builds

File: recipes/rnaz/build.sh (hunk 2 above)
Comment (dpryan79) [reply recipes/rnaz/build.sh]: I've fixed that and used $CFLAGS, which uses -O2

Comment (jfallmann) [reply recipes/rnaz/build.sh]: guys, why is this flag back, as far as I see I removed it with my last commit?

Comment (dpryan79) [reply recipes/rnaz/build.sh]: -O2 is effectively the only one added back, to enable a bit of optimization.

Comment (jfallmann) [reply recipes/rnaz/build.sh]: fine for me, was just wondering why it popped up in the review, thanks for checking

**Comment by dpryan79** [comment]
Thanks @jfallmann!

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**